pages/calldata-decoder.page.tsx

```diff
@@ -92,28 +93,41 @@ export default function CalldataDecoder() {
 
   return (
     <ToolLayout>
-      <h1> Calldata decoder </h1>
-
-      <label htmlFor="calldata" className="text-lg font-bold">
-        Calldata
+      <header className="mb-6 flex items-center gap-3 align-middle">
+        <Img
+          src="/static/svg/decoders.svg"
+          width={32}
+          height={32}
+          alt="deth tools logo"
+        />
+        <h3 className="text-sm text-deth-gray-300 sm:text-xl"> Decoders / </h3>
+        <h3 className="text-sm text-deth-pink sm:text-xl">
+          {' '}
+          Calldata Decoder{' '}
```

Comment:
Redundant spaces attack again!

```suggestion
          Calldata Decoder
```